### PR TITLE
RHDEVDOCS 5960 Errata RN update for GitOps 1.9.4, 1.10.2, 1.11.1 - 1.11 and gitops-docs

### PR DESCRIPTION
**Pull request by mramendi**
Version(s):

please cp to gitops-docs

Issue:

[RHDEVDOCS-5960](https://issues.redhat.com//browse/RHDEVDOCS-5960)

Link to docs preview:

https://73653--ocpdocs-pr.netlify.app/openshift-gitops/latest/release_notes/gitops-release-notes#gitops-release-notes-1-11-1_gitops-release-notes
https://73653--ocpdocs-pr.netlify.app/openshift-gitops/latest/release_notes/gitops-release-notes#gitops-release-notes-1-10-2_gitops-release-notes
https://73653--ocpdocs-pr.netlify.app/openshift-gitops/latest/release_notes/gitops-release-notes#gitops-release-notes-1-9-4_gitops-release-notes

QE review:
- [x] QE has approved this change.

Additional information:

A respin of #73216 with a one-word style correction and correct version destinations - this version is for gitops-docs-1.11 and gitops-docs. See #73216 for Dev and QE approvals.

### CODE REVIEW

File: modules/gitops-release-notes-1-10-2.adoc [hunk 1]
```diff
@@ -8,12 +8,31 @@
...
+$ oc describe deployment gitops-operator-controller-manager -n openshift-operators
+----
+
 [id="fixed-issues-1-10-2_{context}"]
 == Fixed issues
 
 The following issue has been resolved in the current release:
 
-* Before this update, all versions of Argo CD `v2.8.3` and later were vulnerable to cross-server request forgery (CSRF) attacks. As a result, Argo CD would accept non-GET requests even if they did not specify their content type. This update fixes the issue by upgrading the Argo CD to `v.2.8.9` and patching this vulnerability in the Argo CD API. link:https://issues.redhat.com/browse/GITOPS-3922[GITOPS-3922]
+* Before this update, all versions of Argo CD `v2.8.3` and later were vulnerable to cross-server request forgery (CSRF) attacks. As a result, Argo CD would accept non-GET requests even if they did not specify their content type. This update fixes the issue by upgrading the Argo CD version to `v.2.8.9` and patching this vulnerability in the Argo CD API. link:https://issues.redhat.com/browse/GITOPS-3844[GITOPS-3844]
```
Comment (agantony) [inline modules/gitops-release-notes-1-10-2.adoc:35]:
```suggestion
* Before this update, all versions of Argo CD `v2.8.3` and later were vulnerable to cross-server request forgery (CSRF) attacks. As a result, Argo CD accepted non-GET requests even if they did not specify a content type. This update fixes the issue by upgrading the Argo CD version to `v.2.8.9` and patches this vulnerability in the Argo CD API. link:https://issues.redhat.com/browse/GITOPS-3844[GITOPS-3844]
```

File: modules/gitops-release-notes-1-11-1.adoc [hunk 2]
```diff
@@ -8,12 +8,31 @@
...
+[id="rhsa-2024-0689-gitops-1-11-1-security-update-advisory_{context}"]
+=== RHSA-2024-0689 - {gitops-title} 1.11.1 security update advisory
+
+Issued: 2024-02-05
+
+The list of security fixes that are included in this release is documented in the following advisory:
+
+* link:https://access.redhat.com/errata/RHSA-2024:0689[RHSA-2024-0689]
+
+If you have installed the {gitops-title} Operator, run the following command to view the container images in this release:
```
Comment (agantony) [inline modules/gitops-release-notes-1-11-1.adoc:23]:
```suggestion
If you have installed the {gitops-title} Operator, view the container images in this release by running the following command:
```

File: modules/gitops-release-notes-1-11-1.adoc [hunk 3]
```diff
@@ -8,12 +8,31 @@
...
+$ oc describe deployment gitops-operator-controller-manager -n openshift-operators
+----
+
 [id="fixed-issues-1-11-1_{context}"]
 == Fixed issues
 
 The following issue has been resolved in the current release:
 
-* Before this update, all versions of Argo CD `v2.9.2` and later were vulnerable to cross-server request forgery (CSRF) attacks. As a result, Argo CD would accept non-GET requests even if they did not specify their content type. This update fixes the issue by upgrading the Argo CD to `v.2.9.5` and patching this vulnerability in the Argo CD API. link:https://issues.redhat.com/browse/GITOPS-3923[GITOPS-3923]
+* Before this update, all versions of Argo CD `v2.9.2` and later were vulnerable to cross-server request forgery (CSRF) attacks. As a result, Argo CD would accept non-GET requests even if they did not specify their content type. This update fixes the issue by upgrading the Argo CD version to `v.2.9.5` and patching this vulnerability in the Argo CD API. link:https://issues.redhat.com/browse/GITOPS-4219[GITOPS-4219]
```
Comment (agantony) [inline modules/gitops-release-notes-1-11-1.adoc:35]:
```suggestion
* Before this update, all versions of Argo CD `v2.9.2` and later were vulnerable to cross-server request forgery (CSRF) attacks. As a result, Argo CD accepted non-GET requests even if they did not specify a content type. This update fixes the issue by upgrading the Argo CD version to `v.2.9.5` and patches this vulnerability in the Argo CD API. link:https://issues.redhat.com/browse/GITOPS-4219[GITOPS-4219]
```

File: modules/gitops-release-notes-1-9-4.adoc [hunk 4]
```diff
@@ -8,12 +8,31 @@
...
+[id="rhsa-2024-0691-gitops-1-9-4-security-update-advisory_{context}"]
+=== RHSA-2024-0691 - {gitops-title} 1.9.4 security update advisory
+
+Issued: 2024-02-05
+
+The list of security fixes that are included in this release is documented in the following advisory:
+
+* link:https://access.redhat.com/errata/RHSA-2024:0691[RHSA-2024-0691]
+
+If you have installed the {gitops-title} Operator, run the following command to view the container images in this release:
```
Comment (agantony) [inline modules/gitops-release-notes-1-9-4.adoc:23]:
```suggestion
If you have installed the {gitops-title} Operator, view the container images in this release by running the following command:
```

File: modules/gitops-release-notes-1-9-4.adoc [hunk 5]
```diff
@@ -8,12 +8,31 @@
...
+$ oc describe deployment gitops-operator-controller-manager -n openshift-operators
+----
+
 [id="fixed-issues-1-9-4_{context}"]
 == Fixed issues
 
 The following issue has been resolved in the current release:
 
-* Before this update, all versions of Argo CD `v2.7.2` and later were vulnerable to cross-server request forgery (CSRF) attacks. As a result, Argo CD would accept non-GET requests even if they did not specify their content type. This update fixes the issue by upgrading the Argo CD to `v.2.7.16` and patching this vulnerability in the Argo CD API. link:https://issues.redhat.com/browse/GITOPS-3921[GITOPS-3921]
+* Before this update, all versions of Argo CD `v2.7.2` and later were vulnerable to cross-server request forgery (CSRF) attacks. As a result, Argo CD would accept non-GET requests even if they did not specify their content type. This update fixes the issue by upgrading the Argo CD version to `v.2.7.16` and patching this vulnerability in the Argo CD API. link:https://issues.redhat.com/browse/GITOPS-4218[GITOPS-4218]
```
Comment (agantony) [inline modules/gitops-release-notes-1-9-4.adoc:35]: same suggestion as above

**Review by agantony — COMMENTED** [review]
Added a few suggestions for your consideration; overall lgtm!

**Review by JoeAldinger — COMMENTED** [review]
If all of this documentation is being added to previously published docs then this needs to go through CM.

**Pull request merged**